language/diem-framework/modules/DiemAccount.move

```diff
@@ -1937,20 +1951,29 @@ module DiemFramework::DiemAccount {
             Errors::limit_exceeded(PROLOGUE_ESEQUENCE_NUMBER_TOO_BIG)
         );
 
-        // [PCA11]: Check that the transaction sequence number is not too old (in the past)
-        assert(
-            txn_sequence_number >= sender_account.sequence_number,
-            Errors::invalid_argument(PROLOGUE_ESEQUENCE_NUMBER_TOO_OLD)
-        );
+        if (CRSN::has_crsn(transaction_sender)) {
+            // [PCA13]: If using a sequence nonce check that it's accepted
+            assert(
+                CRSN::check(sender, txn_sequence_number),
```

Comment:
@moeziniaf This is the check in the prologue that will not allow the transaction to pass transaction validation (and enter mempool) unless the transaction's sequence nonce is within the window.

Reply:
This means that if there's a window change we have to wait for this window change to commit before we can send new transaction to the system?